src/components/common/DetailsList.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div id="correction-comment">
+    <div class="comments-section mt-8" v-if="filing.comments.length > 0">
+      <v-divider></v-divider>
+      <div class="title-bar mt-5">
+        <h4>
+          <v-icon>mdi-comment-text</v-icon>
+          Detail{{filing.comments.length > 1 ? "s" : ""}} ({{filing.comments.length}})
+        </h4>
+        <v-btn
+          color="primary"
+          v-if="isStaff && !isTask"
```

Comment:
The uxpin shows this button even in the Todo List.
https://preview.uxpin.com/cefa7724eb7ad8538cb92f016364066b202fbec2#/pages/123078669/simulate/no-panels

Reply:
(May no longer need "isTask" prop since I think DetailsList is the same in both Todo List and Filing History List.)